docs/dictionary.rst

```diff
@@ -1,1116 +0,0 @@
-.. DO NOT EDIT THIS FILE. This file is autogenerated from glossary_generator.py, update that instead.
```

Comment:
Seems odd, did the auto-generation fail? Was this not meant to be checked in?
File has a couple years of history, so thinking the former

Reply:
Hmmm, I see the commit comment: 
```
It's a generated file. I did not know this, kept editing it to
make a change, only to see my change get overwritten. Delete
it to avoid future confusion.
```

Is it worth checking that the dictionary is properly auto-generated/displayed by readthedocs without this file checked in?

Reply:
My first commit to the file was done because the generator had changed it. After pushing it, I belatedly realized that would be confusing in the future, so I did a second commit to just delete the file.

> Is it worth checking that the dictionary is properly auto-generated/displayed by readthedocs without this file checked in?

I'll take a look.